browser/ui/webui/brave_tip_ui.cc

```diff
@@ -60,6 +60,9 @@ class TipMessageHandler : public WebUIMessageHandler,
   void OnRecurringTipRemoved(RewardsService* rewards_service,
                              bool success) override;
 
+  void OnPendingContributionSaved(RewardsService* rewards_service,
```

Comment:
Nit: I'm fine with `const` by-value parameters, but I think we agreed on omitting `const` for them.